### DISCUSSION TITLE
plugin to Automate gRPC request

### DISCUSSION THREAD

**Discussion by sivasundar16** [pactumjs/pactum#215] [Q&A]
I am looking for a way to Automate BE services which is build on gRPC. Preferably looking for a NPM plugin to send the request(Method name, body parameters and service details) and receive the response. Please let me know if anyone knows whether pactumjs has a testing tool to test gRPC request like it already has for Rest API. It would be greatly appreciated if any one have the detail or any tool that support gRPC Automation.

**Comment by ASaiAnudeep** [comment]
Hello @sivasundar16, 

As of now gRPC protocol is not supported for testing with pactum. We have a task in our backlog to support it as well.
